rules/successful-reachouts/rule.md

```diff
@@ -0,0 +1,37 @@
+---
+type: rule
+title: Do you know to make your Linkedln reachouts stand out?
+uri: successful-reachouts
+authors:
+  - title: Brady Stroud
+    url: https://www.ssw.com.au/people/brady-stroud
+created: 2024-03-1T01:55:35.941Z
+guid: 012a5f87-5cfb-4002-8d5f-4d2a8787ed6a
+---
+
+Developers receive a lot of reachouts. Sometimes they are automated and other times they are a person copying and pasting the same message to everyone.
+
+To have your reachout rise above the noise, you need to make it personal and unique.
+
+![Figure: Typical developers LinkedIn inbox](developer-reachout-spam.png)
```

Comment:
@bradystroud 

- [ ] 1. Should we make this the "bad" example?
- [ ] 2. Consider making this a medium image... as it's too vertical (causing scrolling) https://ssw.com.au/rules/rule/#6-images